pkgs/applications/networking/powerdns-admin/default.nix

```diff
@@ -91,6 +91,7 @@ in stdenv.mkDerivation rec {
 
   postPatch = ''
     rm -r powerdnsadmin/static powerdnsadmin/assets.py
+    sed -i "s/id:/'id':/" migrations/versions/787bdba9e147_init_db.py
```

Comment:
Why is this required?

Reply:
Without this patch, running the migration fails with a `TypeError: '<' not supported between instances of 'str' and 'builtin_function_or_method'` because `id` is a function instead of a string.